src/FINDOLOGIC/Export/Exceptions/EmptyElementsNotAllowedException.php

```diff
@@ -0,0 +1,12 @@
+<?php
+
+namespace FINDOLOGIC\Export\Exceptions;
+
+class EmptyElementsNotAllowedException extends \RuntimeException
+{
+    public function __construct($elementType, $elementKey)
```

Comment:
Typings?

Reply:
Done! Updated message too so we use a formated string here.